src/device/location/location.android.ts

```diff
@@ -14,8 +14,8 @@ const PriorityAndroidMapping = {
   NO_POWER: 105 // PRIORITY_NO_POWER
 };
 const SETTINGS_STATUS_CODES = {
```

Comment:
Do you think that we need to keep SETTINGS_STATUS_CODES ?

Reply:
The onFailure callback of native provides string parameters (DENIED,  SETTINGS_CHANGE_UNAVAILABLE). 
We are using SETTINGS_STATUS_CODES for mapping string values with LocationAndroidSettingsStatusCodes enum.